auth0/resource_auth0_custom_domain.go

```diff
@@ -47,13 +47,6 @@ func newCustomDomain() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"verification_method": {
```

Comment:
We're removing this as it's been deprecated and no longer accepted by the API (checked the schemas and it's not there any more). This was forcing this resource to always recreate as ForceNew was set to true.